contrib/initramfs/scripts/zfs

```diff
@@ -106,11 +106,11 @@ find_rootfs()
 # Support function to get a list of all pools, separated with ';'
 find_pools()
 {
-	pools=$("$@" 2> /dev/null | \
+	find_pools=$("$@" 2> /dev/null | \
```

Comment:
Technically not part of the original problem, but if we're going to rename parameter variables, better do it everywhere just to be safe and be consistent.